_spinnaker_install_admin_guides/manual-service-account.md

```diff
@@ -12,21 +12,21 @@ redirect_from:
 
 When connecting Spinnaker to Kubernetes, Spinnaker needs the following:
 
-* A service account in the relevant Kubernetes cluster (or namespace in a cluster).  *In Kubernetes, a service account exists in a given namespace, but may have access to other namespaces or to the whole cluster*
+* A service account in the relevant Kubernetes cluster (or namespace in a cluster).  *In Kubernetes, a service account exists in a given namespace but may have access to other namespaces or to the whole cluster*
 * Permissions for the service account to create/read/update/delete objects in the relevant Kubernetes cluster (or namespace)
-* A kubeconfig that has access to the service account, via a token or some other authentication method.
+* A kubeconfig that has access to the service account through a token or some other authentication method.
 
-The [spinnaker-tools binary](https://github.com/armory/spinnaker-tools) was built to create all of the above objects, but if you want to run these manually or need to know what is going on, you can follow these steps to create these objects manually.
+The [spinnaker-tools binary](https://github.com/armory/spinnaker-tools) was built to create all of the above objects. If you want to create these manually or need to know what is going on, use the following steps to create these objects .
 
-**This document primarily uses `kubectl`, and assumes you have access to permissisons that can create and/or update these resources in your Kubernetes cluster:
+**This document primarily uses `kubectl` and assumes you have access to permissions that can create and/or update these resources in your Kubernetes cluster:
 
 * Kubernetes Service Account(s)
 * Kubernetes Roles and Rolebindings
-* (Optionally) Kubernetes ClusterRoles and Rolebindings
+* (Optionally) Kubernetes ClusterRoles and Rolebindings**
```

Comment:
@justinrlee I closed your bold here. It wasn't closed in your commit.  You can also close it before the list if you prefer it that way.